test/node/conversation-items.js

```diff
@@ -137,4 +137,31 @@ describe('Conversation Items', () => {
         const res = await client.getItemById(item.itemId);
         assert(!res.text.likedByUsers || !res.text.likedByUsers.includes(user.userId));
     });
+
+    // it('should update a complex text item and raise an itemUpdated event', async () => {
```

Comment:
why is this commented?

Reply:
You said not to add the file upload attatchment because of needing to require file-api, I guess I should just remove that instead of leaving it in commented?

Reply:
Ok sorry. Just leave it, I'll take a look later.

But you should still keep the "complex" test, just don't do attachments